### PR TITLE
Getting loyalty app ready for prod

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -22,7 +22,22 @@ gulp.task('compile-server', () => {
...
+            "presets": [
+                "es2015",
+                "stage-3",
+                "react"
+            ],
+            "plugins": [
+                "react-relay",
+                ["module-resolver", {
+                    "alias": {
+                        "components": "./dist/components"
```
Comment (alloy) [inline gulpfile.js]: Do you have a real understanding of how the module resolver plugin was failing and how this fixes it?

**Comment by alloy** [comment]
I’d like to not spend too much more time on chasing tails right now and get a release out, so unless you truly understand the module resolver issue, let’s skip that for now as per the original plan #55.

Can you PR the font and package.json changes onto my PR? #97 and we continue there?

**Comment by l2succes** [comment]
Right, I'm with you on getting a release out. Sure, let's skip it for now.
Ok sounds like a plan

**Comment by alloy** [comment]
Ace 👍

**Pull request closed without merging**